mobify/platform-scaffold
# AMP Analytics

Add analytics component that supports all the tracking we have in the prototype. 

 **JIRA**: https://mobify.atlassian.net/browse/AMP-56
 **Linked PRs**: Based on #618 

## Changes
- Include Analytics component that renders appropriate based on provided configuration
- Include Link component so links are instrumented automatically
- Both components should eventually make their way to the SDK

## TODO
- [x] Implement outbound link tracking
- [x] Link component tests

## How to test-drive this PR
- Run `npm run dev`
- Go to http://localhost:3000 and open network tab, observe events sent to EE and GA
- To see the variable substitution working, the server has to use HTTPS...I have changes I can share to enable that. If we think it's worthwhile, I can include the code and self-signed cert into the repo
